src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -544,6 +544,9 @@ export default {
 			if (this.sendingFailure === 'quota') {
 				return t('spreed', 'Not enough free space to upload file')
 			}
+			if (this.sendingFailure === 'failed-share') {
+				return t('spreed', 'You cannot share files')
```

Comment:
why not "You are not allowed to share files" like below ? it's a bit clearer.

Reply:
Done